models/trigger_map_item.go

```diff
@@ -171,53 +308,24 @@ func (triggerItem TriggerMapItemModel) IsDraftPullRequestEnabled() bool {
 	return draftPullRequestEnabled
 }
 
-func (triggerItem TriggerMapItemModel) String(printTarget bool) string {
+func (triggerItem TriggerMapItemModel) String() string {
```

Comment:
possible alternatives: fmt based string representation

Reply:
@godrei or anything that works to make a struct into a stirng representation, for example JSON.